model_api/python/openvino/model_api/models/types.py

```diff
@@ -172,7 +172,7 @@ def __init__(
     def from_str(self, value):
         try:
             floats = [float(i) for i in value.split()]
-            ints = [int(i) for i in value.split()]
+            ints = [int(float(i)) for i in value.split()]
```

Comment:
not sure if I understand this? What's the motivation here?

Reply:
When the string is something like this `"7.8 4.1 3.2"`, performing conversion like `int('7.8')` throws an exception. String to int conversion requires that the string does not contain decimal values.

Reply:
Thanks! I think this would be a nice comment in the code :D